### ISSUE TITLE
ManytoMany field with many registers

### ISSUE THREAD

**Issue by elpipesalazar** [sshwsfc/django-xadmin#252]
I have two models which have a relationship. I need to show this in admin page, but i don't want use the default widget for this. Because every model have many registers and this causa a lag on my admin page. How can i show this relationship? any ideas?

"""This is my main model. (Related with Event model)"""
class Artist(models.Model):
   ...
   events = models.ManyToManyField(Event, related_name='artists')

**Comment by jantos** [comment]
Hi,

Are you maybe looking for something like this:
adminx.py
  class EventAdmin(object):
    style_fields = {'events': 'm2m_transfer'}

The above code shows the same ManyToMany selection box that you can see under admin -> users -> Permissions.

**Comment by elpipesalazar** [comment]
Yes. But the problem is when the related model there are thousands of records. This makes it very slow load

**Comment by light4** [comment]
`raw_id_fields` does not effect on ManyToManyField. So on this condition, what should we do?
